src/DynamoCore/Search/SearchElements/NodeModelSearchElement.cs

```diff
@@ -12,19 +12,25 @@ namespace Dynamo.Search.SearchElements
     public class NodeModelSearchElement : NodeModelSearchElementBase
     {
         private readonly Func<NodeModel> constructor;
+        private readonly string creationName;
 
+        public override string CreationName { get {return creationName;} }
+        
```

Comment:
@aparajit-pratap I think this change is fine, we'll just need to make sure we deal with any changes to the CreationName in the client. If / when do you merge this can update this task tracking this change:
https://jira.autodesk.com/secure/RapidBoard.jspa?rapidView=5034&projectKey=QNTM&view=planning&selectedIssue=QNTM-1232 so we know to address it then.